### PR TITLE
add the instructions to create the .po file

### PR DESCRIPTION
**Pull request by duckish**
fix#48

### CODE REVIEW

File: docs/creat_po_file.adoc [hunk 1]
```diff
@@ -0,0 +1,17 @@
+## instructions
+
+1) In your model create a folder "locale"
```
Comment (RignonNoel) [inline docs/creat_po_file.adoc]: It's in the application, not in the model. The model is a file not a directory

File: docs/creat_po_file.adoc [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+2) In "settings.py" add the locale path in "LOCALE_PATHS"
+
+    for exemple
+
+    LOCALE_PATHS = (
```
Comment (RignonNoel) [inline docs/creat_po_file.adoc]: Use ` to set code in format.

**Comment by RignonNoel** [comment]
Be carefull your name of file is: ```docs/creat_po_file.adoc```

There is a letter missing

**Comment by maximest-pierre** [comment]
I co-wrote the change with duc can someone please check it @RignonNoel  @manumilou

**Comment by RignonNoel** [comment]
C'est pas mal pour moi.

Peut être juste rajouter un lien vers la documentation officiel du makemessage? https://docs.djangoproject.com/en/1.9/ref/django-admin/#makemessages

Mais c'est facultatif...

**Pull request merged**